docs/Technical-Documentation/django-admin-logging.md

```diff
@@ -0,0 +1,51 @@
+# Django Admin Logging
```

Comment:
@ADPennington let me know if there is anything else you see fit to include in here.